Contracts/Contracts.Tests/TesteContracteTerti/TestData.cs

```diff
@@ -38,6 +40,29 @@ public static ContractTerti GenerateContractTerti(string id)
                     }
                     },
                     StateDeFunctii = new StateDeFunctii()
+                    {
+                        SemnaturaDirectorDepartament = true,
+                        SemnaturaResponsabilContract = false,
+                        StatDeFunctiiList = new List<StatDeFunctii>
+                        {
+                            new StatDeFunctii
+                            {
+                                Id = "05972b6c-5104-4723-8c6f-6f4a3b65e35e",
+                                Nume = "Mircea",
+                                GradDidactic = "Profesor",
+                                DataIncepriiActivitatii = "12/05/2018",
```

Comment:
DataIncepriiActivitatii/DataFinalizariiActivitatii are tipul string, ar trebui sa fie DateTime si atunci poti sa-i setezi valoarea asa
```suggestion
                                DataIncepriiActivitatii = DateTime.Parse("12/05/2018"),
```